util/conan-package/.gitignore

```diff
@@ -15,4 +15,5 @@ test_package/build
 conanfile.pyc
 conaninfo.txt
 conanbuildinfo.txt
-conanbuildinfo.cmake
\ No newline at end of file
+conanbuildinfo.cmake
+!FindGLM.cmake
```

Comment:
There is an exclusion of all .cmake files in the .gitignore of the top level directory:

https://github.com/g-truc/glm/blob/master/.gitignore